scenarios/security_audit/rand_webapp.xml

```diff
@@ -98,19 +98,19 @@
     </build>
   </system>
 
-  <system>
```

Comment:
Not sure if you meant to remove Kali from the scenario, or if that's just for testing

Reply:
Thanks, Cliffe. I  commented the Kali  provisioning out temporarily as it was causing the build to fail. Error says it was unable to locate the 'kali-tools-web' package. I believe the repository for this dependency may need to be added to the project.

Reply:
I logged it as ticket #199 as it is a separate issue.